spec/models/food_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  subject do
+    @user = User.create(name: 'Faranosh', email: 'faranosh@mail.com', password: 'password')
+    Food.create(name: 'Corn', measurement_unit: 'grams', price: 2, user: @user)
+  end
+
+  before(:each) { subject.save }
+
+  it 'should have name corn' do
+    expect(subject.name).to eql('Corn')
+  end
+
+  it 'should have Measurement unit grams' do
+    expect(subject.measurement_unit).to eql('grams')
+  end
+
+  it 'Should always have Measurement unit present' do
+    subject.measurement_unit = nil
+    expect(subject).to_not be_valid
```

Comment:
- [ ] Good job working on the test for models but I think you have some failed tests as well since you have 8 tests are not passed si kindly work on them to let them pass, you may need to fix some issues in the models itself to let them pass so kindly work on them to let all your tests run properly 🔥💯⭐✅



![image](https://github.com/Tafloninno/recipe-app/assets/58553711/03cac828-2db2-4469-b669-66ddf8137e3a)

Reply:
@Reem-lab Thanks for the kind review